src/libraries/Microsoft.PowerFx.Core/Public/Values/DateTimeValue.cs

```diff
@@ -23,5 +23,10 @@ public override void Visit(IValueVisitor visitor)
         {
             visitor.Visit(this);
         }
+
+        public override string ToExpression()
+        {
+            return $"DateTime({Value.ToString("yyyy,M,d,H,m,s")})";
```

Comment:
>Value.ToString("yyyy,M,d,H,m,s [](http://example.com/codeflow?start=31&length=30)

This expression depends on the local locale and would result in "2022,9,23,18,46,24" when in French.

You should convert to a neutral format (ISO 8601) and UTC

if (dt.Kind != DateTimeKind.Utc)
 dt = dt.ToUniversalTime();
dt.ToString("o")

This gives "2022-09-23T16:49:02.1901316Z"

Reply:
That's a very good point - I think the challenge is that we don't have a good way to deserialize ISO - which we should.

Reply:
DateTimeValue(ISO value)

Reply:
After discussion, I think just naively serializing the DateTime is sufficient.  That will include the timezone specifier and will deserialize faithfully. 

Our primary serialization scenario is deserializing across multiple requests to a server - so the target timezone will be constant.